### DISCUSSION TITLE
How to select?

### DISCUSSION THREAD

**Discussion by foco76** [LSPosed/MagiskOnWSALocal#215] [Q&A] [answered]
When i run build.sh it automatically select like this-
INFO: Architecture: x64
INFO: Release Type: retail
INFO: Magisk Version: stable
INFO: GApps Brand: MindTheGapps
INFO: GApps Variant: pico
INFO: Root Solution: magisk
Build: RELEASE_TYPE=Retail
 i can't choose things way i want!how to fix this?

**Comment by Howard20181** [comment, marked as answer]
`build.sh` is a command line program, entered via parameters.
Run `run.sh`

**Comment by foco76** [comment]
it stop from here-
Checking and ensuring dependencies
[sudo] password for builder: 
Hit:1 https://dl.winehq.org/wine-builds/debian bullseye InRelease                                           
Hit:3 https://deb.parrot.sh/parrot parrot InRelease                                                         
Hit:4 https://deb.parrot.sh/direct/parrot parrot-security InRelease
Hit:5 https://deb.parrot.sh/parrot parrot-backports InRelease
Hit:2 https://deb.parrot.sh/parrot lts-updates InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
e2fsprogs is already the newest version (1.46.2-2).
The following packages were automatically installed and are no longer required:
  libcapi20-3 libfaudio0 libstb0 libvkd3d1 libwine wine64
Use 'sudo apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 45 not upgraded.

**Reply by Howard20181** [reply]
It looks like there is no error

**Comment by foco76** [comment]
yes..there is none..but the problem is..i can't choose variants in gapps and magisk versions is there way to edit them in build.sh? by manually typing?

**Reply by Howard20181** [reply]
Check that your wine installation is working
Run `wine cmd` to see if the built-in cmd can be invoked properly

**Reply by foco76** [reply]
Never mind..I figure out the way..but thank you very much...
For others if you ran into this..
change variants in in  default() {xxxx} -Line 95
Good Luck!
